docs/reference/commandline/rmi.md

```diff
@@ -26,6 +26,17 @@ Options:
       --no-prune   Do not delete untagged parents
 ```
 
+## Description
+
+Removes (and un-tags) one or more images from the host node. If an image has
+multiple tags, using this command with the tag as a parameter only removes the
+tag. If the tag is the only one for the image, both the image and the tag are
+removed.
+
+This does not remove images from a registry. You cannot remove an image of a
+running container unless you use the `-f` option. To see all images on a host
+use the [`docker image ls`](images.md) command.
```

Comment:
Can you add a small paragraph about the `--no-prune` flag too? I think its behavior is not that obvious regarding its name and then deserve some explanation 😄 .

Reply:
@silvin-lubecki there's definitely more improvements to be made to this page; this PR was only intended to port the changes made in https://github.com/docker/cli/pull/1444 to the online documentation, so I'd rather keep that for a separate change (I'll see if I can make some time for additional improvements).

For that follow-up; if we _do_ add that description, it should likely go into the `examples` section.